src/map/itemdb.cpp

```diff
@@ -439,7 +439,7 @@ static void itemdb_create_dummy(void) {
 	CREATE(dummy_item, struct item_data, 1);
 
 	memset(dummy_item, 0, sizeof(struct item_data));
-	dummy_item->nameid = 500;
+	dummy_item->nameid = 499;
```

Comment:
Can you please move the declaration to itemdb.hpp?
And call it `ITEMID_DUMMY`.
Thank you!

Reply:
Done. Nice tip!